sdk/textanalytics/ai-text-analytics/src/textAnalyticsResult.ts

```diff
@@ -87,6 +91,40 @@ export interface TextAnalyticsErrorResult {
   readonly error: TextAnalyticsError;
 }
 
+export interface TextAnalyticsResultArray<T1 extends TextAnalyticsSuccessResult>
```

Comment:
`T1` is a strange name from a generic. `TSuccess` perhaps would be more meaningful?

Reply:
Done.